lib/livebook/config.ex

```diff
@@ -302,6 +302,16 @@ defmodule Livebook.Config do
     module not in @identity_provider_no_id
   end
 
+  @doc """
+  Returns metadata of a ZTA provider
+  """
+  @spec zta_metadata(atom()) :: nil | map()
+  def zta_metadata(nil), do: nil
```

Comment:
I think this function shouldn't expect nil, rather the caller should conditionally handle nil and call this function only with a valid type :)

Reply:
Makes sense. Changed in https://github.com/livebook-dev/livebook/pull/2572/commits/b5003072d5070d3bcb93f8adb88decbc0b5eb271.